libs/newsletters-data-client/src/lib/schemas/newsletter-data-type.ts

```diff
@@ -163,9 +163,11 @@ export const newsletterDataSchema = z.object({
 	brazeCampaignCreationStatus: workflowStatusEnumSchema.describe(
 		'Braze campaign creation status',
 	),
-	ophanCampaignCreationStatus: workflowStatusEnumSchema.describe(
-		'Ophan campaign creation status',
-	),
+	ophanCampaignCreationStatus: workflowStatusEnumSchema
+		.optional()
+		.describe(
+			'Ophan campaign creation status',
```

Comment:
maybe add a comment or update the field description to mark the property as deprecated?

Reply:
Yes, great idea. Thanks